src/include/platform/OTARequestorDriver.h

```diff
@@ -35,6 +35,7 @@ struct UpdateDescription
 {
     CharSpan imageURI;
     uint32_t softwareVersion;
+    CharSpan softwareVersionStr;
```

Comment:
This is a non owning view, is there a guaranty that it won't outlive the response ?

Reply:
No, the description update object is created locally in the OTA query image response callback and uses pointers to response elements, so everything is go away with the response object. But... UpdateAvailable() callback is called inside the OTA query image response callback, so there is a guarantee in this case.